README.md

```diff
@@ -263,6 +263,18 @@ The default GitHub Issues and Pull Requests interface.
 - Nested documents support in Grafana (Network, CPU, Disks, ..). Refer to [PR #4694 of grafana](https://github.com/grafana/grafana/pull/4694) for more.
 - Timeshift feature to compare 2 different sa binaries
 
+### Setting up development environment
+
+- Clone the repo
+- run cp ``` cp env.example .env ```
+- Start the containers with ``` docker compose up ```
+- Kill the web container ``` docker kill sarjitsu_web_1 ```
+- Navigate to ``` sarjitsu/lib/backend/src ```
+- Install dependencies with ``` sudo dnf install python3-devel && pip install -r requirements.txt ```
+- Modify CFG_PATH in config.py to ``` lib/backend/conf/sarjustu.ini.example ```
```

Comment:
wouldn't that relative path depend on where we execute `python server.py` from ? more like an indication to user to set this per their env?